microsoft365defender_consts.py

```diff
@@ -126,6 +126,8 @@
 DEFENDER_NO_ALERT_FOUND = "No alert found"
 DEFENDER_NO_INCIDENT_FOUND = "No incident found"
 DEFENDER_ALERT_UPDATED_SUCCESSFULLY = "Successfully updated the alert"
+DEFENDER_INCIDENT_UPDATED_SUCCESSFULLY = "Successfully updated the incident"
+DEFENDER_INCIDENT_NO_PARAMETER_PROVIDED = "Please provide at least one of the property to update the incident"
```

Comment:
"one of the property"? Should it be "one of the properties"?

Reply:
Done